grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -30,21 +30,22 @@ class OrderController {
         redirect(action: "list", params: params)
     }
 
-    def list = {
+    def list = { OrderCommand command ->
 
         def suppliers = orderService.getSuppliers().sort()
 
-        def name = params.name
-        def orderNumber = params.orderNumber
-        def orderTypeCode = params.orderTypeCode ? params.orderTypeCode as OrderTypeCode : null
-        def origin = params.origin ? Location.get(params.origin) : null
-        def destination = params.destination ? Location.get(params.destination) : Location.get(session?.warehouse?.id)
-        def status = params.status ? Enum.valueOf(OrderStatus.class, params.status) : null
+        def orderTemplate = new Order(params)
+        orderTemplate.status = null
+//        def name = params.name
```

Comment:
Do we need this code left here?

Reply:
Thanks for the reminder. I meant to test to make sure all of those parameters were working before I removed them. Should be good now.